src/helix-ui/elements/HXFormControlElement.js

```diff
@@ -108,14 +116,32 @@ export class HXFormControlElement extends HXElement {
     /** @private */
     _onCtrlBlur () {
         // communicate state via read-only, boolean content attributes
-        this.$defaultAttribute(STATE.touched, '');
-        this.$defaultAttribute(STATE.dirty, '');
+        this._stateTouched();
+        this._stateDirty();
     }
 
     /** @private */
     _onCtrlChange () {
         // communicate state via read-only, boolean content attributes
+        this._stateChanged();
+        this._stateDirty();
+    }
+
+    /** @private */
+    _stateChanged () {
         this.$defaultAttribute(STATE.changed, '');
+        this.$emit('hxchanged', { bubbles: true });
+    }
+
+    /** @private */
+    _stateDirty () {
         this.$defaultAttribute(STATE.dirty, '');
+        this.$emit('hxdirty', { bubbles: true });
+    }
+
+    /** @private */
+    _stateTouched () {
+        this.$defaultAttribute(STATE.touched, '');
+        this.$emit('hxtouched', { bubbles: true });
```

Comment:
Let's modify the tense of the event to `hxtouch` (similar to previous comment).

Reply:
modified to `hxtouch` event